File: projekat/src/app/entities/company/company.ts
import { Rating } from '../rating/rating';

export class Company {
    name: string;
    description: string;
    logo: string;
    rating: Rating;

    constructor(name: string, description: string, logo: string){
        this.description = description;
        this.name = name;
        this.logo = logo;
        this.rating = new Rating();
    }
}

File: BookingService/BookingAPI/Model/Company.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace BookingAPI.Model
{
    public class Company
    {
        [Key]
        public int Id { get; set; }
        public string AdminUserName { get; set; }
        [Required]
        public string Name { get; set; }
        public string Description { get; set; }
        public Image Logo { get; set; }
    }
}
